README.md

```diff
@@ -8,34 +8,19 @@
 [![Coverage Status](https://codecov.io/gh/aws/aws-sdk-js/branch/master/graph/badge.svg)](https://codecov.io/gh/aws/aws-sdk-js)
 [![Known Vulnerabilities](https://snyk.io/test/github/aws/aws-sdk-js/badge.svg)](https://snyk.io/test/github/aws/aws-sdk-js)
 
-**Note**: The [version 3.x](https://github.com/aws/aws-sdk-js-v3) of the AWS SDK for JavaScript is generally available. For more information see the [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/) or [API Reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html).
+## VERSION 3.x NOW AVAILABLE
+The [version 3.x](https://github.com/aws/aws-sdk-js-v3) of the AWS SDK for JavaScript is generally available. For more information see the [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/) or [API Reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html).
 
 For release notes, see the [CHANGELOG](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md). Prior to v2.4.8, release notes can be found at https://aws.amazon.com/releasenotes/?tag=releasenotes%23keywords%23javascript
 
-If you are upgrading from 1.x to 2.0 of the SDK, please see the
-[upgrading notes](https://github.com/aws/aws-sdk-js/blob/master/UPGRADING.md)
-for information on how to migrate existing code to work with the new major
-version.
+Jump To:
+* [Getting Started](_#Getting-Started_)
+* [Getting Help](_#Getting-Help_)
+* [Contributing](_#Contributing_)
 
-## Installing
+# Getting Started
```

Comment:
Heading level 1 should only be used for title.
Ditto to add other heading levels in other sections of README.
```suggestion
## Getting Started
```

Reply:
Changed a few